client/src/components/HowToPlayComponent/HowToPlay.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
```

Comment:
Let's just create a page, and link this component there as it is just a static list of information and a page with different route.